.github/workflows/ci.yml

```diff
@@ -27,5 +27,5 @@ jobs:
       - name: Install Yarn 🚧
         if: ${{ env.ACT }}
         run: npm install -g yarn
-      - name: Run Tests 🦺
-        run: yarn test
+      - name: Run Tests and Build 🦺
```

Comment:
Why do we want to build here?

Reply:
Want to make sure the generated package is installable as part of the test.
That test is only possible if the package is built, so I've implemented it as part of the `build` task:
https://github.com/rangle/radius-tracker/pull/10/files#diff-67c4aaef15040e0527e1b1955306c8cff66858840340a94b9a5d13752d4d99abR29